stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/ChatTheme.kt

```diff
@@ -132,6 +143,10 @@ public fun ChatTheme(
     imageLoaderFactory: StreamCoilImageLoaderFactory = StreamCoilImageLoaderFactory.defaultFactory(),
     messageAlignmentProvider: MessageAlignmentProvider = MessageAlignmentProvider.defaultMessageAlignmentProvider(),
     messageOptionsUserReactionAlignment: MessageOptionsUserReactionAlignment = MessageOptionsUserReactionAlignment.END,
+    permissionHandlers: List<PermissionHandler> = StreamPermissionHandlers.defaultHandlers(
```

Comment:
We need to reflect this change by updating the docs contained in `01-chat-theme.mdx`